src/main/features/claimant-response/views/check-and-send.njk

```diff
@@ -96,6 +96,66 @@
 
     {{ tableEnd() }}
 
+    {% set courtDetermination = draft.courtDetermination %}
+    {% set courtDecisionPaymentIntention = courtDetermination.courtDecision %}
+
+    {% if courtDetermination and courtDecisionPaymentIntention %}
```

Comment:
As discussed, please add a route test for this section that verifies that it shows when it is supposed too.